internal/daemon/controller/controller.go

```diff
@@ -119,6 +120,8 @@ type Controller struct {
 	apiGrpcServerListener grpcServerListener
 	apiGrpcGatewayTicket  string
 
+	ratelimiter *rate.Limiter
```

Comment:
```suggestion
	rateLimiter *rate.Limiter
```

Reply:
Thanks, fixed in [03ef8dd](https://github.com/hashicorp/boundary/pull/3825/commits/03ef8ddb690b60ba9a01bf9e05021c321e373b53)